### DISCUSSION TITLE
3-way admixtures

### DISCUSSION THREAD

**Discussion by d1eone** [Atkinson-Lab/Tractor#4] [Q&A] [answered]
Does Tractor work with 3-way ancestral admixture, as is seen in Central and South America? I have a population that is admixed with Indigenous American, African, and European populations.

**Comment by eatkinson** [comment, marked as answer]
Yes! Our updated code can handle as many way admixed as you need. The user supplies the number of component ancestries with the --num-ancs flag in the ExtractTracts.py script.
